exe/ExampleMain.hs

```diff
@@ -130,11 +134,29 @@ parseHandler tzs = do
         options = Options {withLatent = fromMaybe False (Main.latent request)}
         parseDimensionList = mapMaybe fromName (parseDimensions request)
         filterDimensionSet = HashSet.fromList (filterDimensions request)
-
-        parsedResult = map
-          (parseDocument options thisLocale tzs parseDimensionList filterDimensionSet)
-          (zip (zip (timezones request) (referenceTimes request)) (texts request))
-
+        configuredParse = parseDocument options thisLocale tzs parseDimensionList filterDimensionSet
+        documents = (zip (zip (timezones request) (referenceTimes request)) (texts request))
+
+        emptyEntitiesOnTimeout :: ((Text, Integer), Text) -> Maybe [Entity] -> Snap [Entity]
+        emptyEntitiesOnTimeout (_, text) Nothing = do
+          logError
+            $ Text.encodeUtf8
+            $ Text.concat [
+              "Timed out when parsing entities in document: ```"
+              , text
+              , "```."
+            ]
+          return []
```

Comment:
FYI, `logError` appears to evaluate character escape sequences here - in the example I ran, it evaluated newlines like `\n\n` in the JSON request and split the log over several lines. Not sure if this is what you intended.

Reply:
Not what I intended no, but I also don't know how to fix it---I'll have a look, but if not, I'll just have to live with it.

Reply:
Right, so the problem isn't that it evaluates newlines in the JSON request, but that the JSON request itself has newline chars. I changed the code to do `show text` (the equivalent of `repr`) in Python.